index.html

```diff
@@ -1104,6 +1104,70 @@ <h2>Extensibility</h2>
 
       </section>
 
+
+      <section>
+        <h2>Caching</h2>
+
+        <p>
+<a>Issuers</a> may wish to signal information related to caching a
+<a>credential</a> to <a>holders</a> and <a>verifiers</a>. Some issuers may
+assert that they do not want a credential to be cached, while others may
+specify certain caching rules and refresh time periods that are acceptable.
+        </p>
+
+        <p>
+Caching information for the credential in the Verifiable Credentials Data
+Model is provided by adding the following <a>properties</a>:
+        </p>
+
+        <dl>
+          <dt><var>cacheControl</var></dt>
+          <dd>
+The value of this property MUST be one or more of the acceptable values for
+the Cache-Control header in HTTP [[RFC7234]] Section
+<a href="https://tools.ietf.org/html/rfc7234#section-5.2">5.2. Cache-Control</a>.
+          </dd>
+          <dt><var>cacheRefreshService</var></dt>
+          <dd>
+The value of this property MUST be a URL. The URL MUST be constructed such that
+an HTTP GET with a query parameter of <code>id</code> and the value of the
+credential `id` property appended will result in an updated credential or an
+error.
```

Comment:
I think the spec should avoid arbitrary URL construction rules like this.  Another option is to use a [URI Template](https://tools.ietf.org/html/rfc6570) with an input of an `id`.  Perhaps an even better option is to just avoid such issues and let the credential specify the full refresh URL in whatever format is appropriate in a `cacheRefreshUrl` property or similar.

Reply:
Ok, agreed. Changing it now.